### DISCUSSION TITLE
Naming conventions of files

### DISCUSSION THREAD

**Discussion by Nimesh-Srivastava** [Nimesh-Srivastava/DSA#17] [Announcements]
## Hello fellow contributors 👋🏼 
As we now have questions from multiple platforms in multiple languages, it is necessary to properly name your file in order to maintain a readable repository.
<br>

Guiding points :
* In the home repository, all questions are grouped by languages, e.g., C, C++, etc.
* In the next level, comes the name of the platform, i.e., LeetCode, Codeforces, CodeChef, etc.
<br>

So, a typical file name should look like : <b>[Repo name] / [Language] / [Platform] / [name].[extension]</b>

For example :
![Image 27-10-21 at 6 26 PM](https://user-images.githubusercontent.com/30381993/139070225-8f1aba60-c7e5-4a33-8e34-3b1a4110a093.jpg)

<br>
That's it. Just keep this in mind and everything should go on smoothly ✌🏼 

## Note
As of already existing files following old naming conventions, I will update them, some files at a time. However, if you want to change your file right away on your own, go ahead and do so. I'll verify and merge the pull request.

**Comment by spycedcoffee** [comment]
Okay!!!

**Comment by Nimesh-Srivastava** [comment]
## Update :octocat:  
All existing code files have now been put in their new directories 🎉 🎉.
